node/cmd/guardiand/node.go

```diff
@@ -1137,81 +1140,91 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 		if shouldStart(avalancheRPC) {
 			logger.Info("Starting Avalanche watcher")
-			readiness.RegisterComponent(common.ReadinessAvalancheSyncing)
+			readinessSync := common.ChainIdToReadinessSyncing(vaa.ChainIDAvalanche)
+			readiness.RegisterComponent(readinessSync)
 			chainObsvReqC[vaa.ChainIDAvalanche] = make(chan *gossipv1.ObservationRequest, observationRequestBufferSize)
 			if err := supervisor.Run(ctx, "avalanchewatch",
-				common.WrapWithScissors(evm.NewEthWatcher(*avalancheRPC, avalancheContractAddr, "avalanche", common.ReadinessAvalancheSyncing, vaa.ChainIDAvalanche, chainMsgC[vaa.ChainIDAvalanche], nil, chainObsvReqC[vaa.ChainIDAvalanche], *unsafeDevMode).Run, "avalanchewatch")); err != nil {
+				common.WrapWithScissors(evm.NewEthWatcher(*avalancheRPC, avalancheContractAddr, "avalanche", readinessSync, vaa.ChainIDAvalanche, chainMsgC[vaa.ChainIDAvalanche], nil, chainObsvReqC[vaa.ChainIDAvalanche], *unsafeDevMode).Run, "avalanchewatch")); err != nil {
 				return err
 			}
 		}
 		if shouldStart(oasisRPC) {
 			logger.Info("Starting Oasis watcher")
+			readinessSync := common.ChainIdToReadinessSyncing(vaa.ChainIDOasis)
+			readiness.RegisterComponent(readinessSync)
```

Comment:
Did oasis not register a component previously?  Or is it somewhere else?

Reply:
It was not there before. I assumed it was an oversight. @evan-gray do you know if it was intentional that Oasis did not register for readiness?